src/common/snippets/src/generator.cpp

```diff
@@ -68,14 +73,15 @@ ngraph::snippets::code ngraph::snippets::Generator::generate(std::shared_ptr<ov:
     for (auto n : m_scalar->get_ordered_ops()) {
         scalar_lowered.emplace_back(std::make_pair(target->get(n->get_type_info())(n), ngraph::snippets::getRegisters(n)));
     }
-    OV_ITT_TASK_NEXT(GENERATE, "::Tiles1D")
+    OV_ITT_TASK_NEXT(GENERATE, "::Tiles1D");
     // wrapping into tiles1D
-    const auto& vector_tile = std::make_shared<ngraph::snippets::op::Tile>(lowered);
+    //todo: in, out, and io_last_dims should derive naturally from the graph representation
```

Comment:
Are you saying about parameters extracted directly from body? If yes how it maps on idea to have planar Tile (in that case it will not have body at all)?

Reply:
Yes, we indeed need planar tiles to do that. It's just how the TileStart should work:
a. If TileStart inputs has > 1 dims (outer tile) then pop_front shapes until TileEnd (for example: [16, 2, 8] -> [2, 8], WA=16)
b. If TileStart inputs has == 1 dims (inner tile) thats exactly our out io_last_dims and we are ready to emit body.